kie-wb/kie-wb-webapp/src/main/java/org/kie/workbench/backend/DashbuilderBootstrap.java

```diff
@@ -47,17 +48,31 @@
     @Inject
     protected DataSetDefRegistry dataSetDefRegistry;
 
+    @Inject
+    protected EntityManagerFactory emf;
+
     @PostConstruct
     protected void init() {
+        // figure out data source JNDI name
+        Object ds = emf.getProperties().get("hibernate.connection.datasource");
+        if (ds != null && ds instanceof javax.naming.Referenceable) {
+            try {
+                jbpmDatasource = ((javax.naming.Referenceable) ds).getReference().getClassName();
+            } catch (Exception e) {
```

Comment:
What evil things will happen if this actually fails? Just thinking if we need to log or rethrow.